resources/CICD/fastlane/Fastfile

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+require './Constants/Constants'
+require './Managers/VersioningManager'
+require './Managers/BuildManager'
+require './Managers/SymbolManager'
+require './Managers/DistributionManager'
+
+versioning_manager = VersioningManager.new(
+  fastlane: self,
+  project_path: Constants.PROJECT_PATH,
+  main_target_name: Constants.MAIN_TARGET_NAME
+)
+
+symbol_manager = SymbolManager.new(
+  fastlane: self,
+  version: versioning_manager.version_number,
+  build_number: versioning_manager.build_number,
+  build_path: Constants.BUILD_PATH,
+  gsp_directory: Constants.GSP_DIRECTORY,
+  binary_path: Constants.BINARY_PATH
+)
+
+builder = BuildManager.new(
+  fastlane: self,
+  scheme_name_staging: Constants.SCHEME_NAME_STAGING,
+  scheme_name_production: Constants.SCHEME_NAME_PRODUCTION,
+  bundle_identifier_staging: Constants.BUNDLE_ID_STAGING,
+  product_name_staging: Constants.PRODUCT_NAME_STAGING,
+  product_name_staging_test_flight: Constants.PRODUCT_NAME_STAGING_TEST_FLIGHT,
+  product_name_production: Constants.PRODUCT_NAME_PRODUCTION
+)
+
+distribution_manager = DistributionManager.new(
+  fastlane: self,
+  build_path: Constants.BUILD_PATH,
+  firebase_token: ENV['FIREBASE_CLI_TOKEN']
+)
+
+keychain_name = "github_action_keychain"
+keychain_password = "password"
+
+before_all do
+  ensure_bundle_exec
+end
+
+default_platform(:ios)
+
+platform :ios do
+
+  desc "Sync development match signing"
+  lane :syncDevelopmentSigning do
```

Comment:
I would prefer `snake_case` style instead of camelCase, we are writing ruby code :D